cockatrice/src/cardfilter.cpp

```diff
@@ -1,47 +1,47 @@
 #include "cardfilter.h"
 
-const char *CardFilter::typeName(Type t)
+const QString CardFilter::typeName(Type t)
 {
     switch (t) {
         case TypeAnd:
-            return "AND";
+            return QString("AND");
```

Comment:
Should these not be translated?

Reply:
I guess this is a good idea; maybe it's not easy to find short, meaningful terms for these logical operators for every language out there, but let translators decide is probably the best choice.